doc-content/enterprise-only/openshift/offline-repo-proc.adoc

```diff
@@ -61,7 +61,7 @@ endif::offline_onprem[]
 +
 [subs="attributes,verbatim,macros"]
 ----
-java -jar offliner-<version>.jar -r https://maven.repository.redhat.com/ga/ -d /home/user/temp {PRODUCT_FILE}-offliner.txt
+java -jar offliner-<version>.jar -r https://maven.repository.redhat.com/ga/ -r https://repo1.maven.org/maven2/ -d /home/user/temp {PRODUCT_FILE}-offliner.txt
 ----
 +
 Replace `/home/user/temp` with an empty temporary directory and `<version>` with the version of the Offliner tool.
```

Comment:
just an nitpick: I would probably mention ot say ` with the latest version of Offliner tool`

Reply:
Would be probably good to do it in the download step too.

Reply:
In the download step, "latest version of the offliner tool" is already mentioned, so here I added the version "that you downloaded"